### PR TITLE
Remove customer returns backend

### PR DESCRIPTION
**Pull request by mbj**
Extraction from #223.

Depends on: #225

### CODE REVIEW

File: backend/app/models/spree/backend_configuration.rb [hunk 1]
```diff
@@ -3,16 +3,14 @@ class BackendConfiguration < Preferences::Configuration
     preference :locale, :string, default: Rails.application.config.i18n.default_locale
 
     ORDER_TABS         ||= [:orders, :payments, :creditcard_payments,
-                            :shipments, :credit_cards, :return_authorizations,
-                            :customer_returns, :adjustments, :customer_details]
+                            :shipments, :credit_cards, :adjustments, :customer_details]
```
Comment (dkubb) [inline backend/app/models/spree/backend_configuration.rb]: Should this list use `%i[...]`?

File: backend/app/models/spree/backend_configuration.rb [hunk 2]
```diff
@@ -3,16 +3,14 @@ class BackendConfiguration < Preferences::Configuration
...
     PRODUCT_TABS       ||= [:products, :option_types, :properties, :prototypes,
                             :variants, :product_properties]
     REPORT_TABS        ||= [:reports]
     CONFIGURATION_TABS ||= [:configurations, :general_settings, :tax_categories,
                             :tax_rates, :payment_methods, :shipping_methods,
                             :shipping_categories,
-                            :stock_locations, :trackers, :refund_reasons,
-                            :reimbursement_types, :return_authorization_reasons]
+                            :stock_locations, :trackers]
```
Comment (dkubb) [inline backend/app/models/spree/backend_configuration.rb]: Instance of https://github.com/MountainRoseHerbs/spree/pull/224/files#r61317573

Comment (mbj) [inline backend/app/models/spree/backend_configuration.rb]: I'll extract this to a dedicated PR fixing the whole file.

File: backend/app/models/spree/backend_configuration.rb (hunk 1 above)
Comment (dkubb) [inline backend/app/models/spree/backend_configuration.rb]: Also, is it valid to use `||=` on a constant? That seems a little weird.

Comment (mbj) [inline backend/app/models/spree/backend_configuration.rb]: Likely this is an artifact for a fix for rails autoloading. Lets kill it.

File: backend/app/views/spree/admin/payments/_list.html.erb [hunk 3]
```diff
@@ -21,11 +21,7 @@
         <td class="align-center"> <span class="state <%= payment.state %>"><%= Spree.t(payment.state, :scope => :payment_states, :default => payment.state.capitalize) %></span></td>
         <td class="actions">
           <% payment.actions.each do |action| %>
-            <% if action == 'credit' %>
-              <%= link_to_with_icon 'reply', Spree.t(:refund), new_admin_order_payment_refund_path(@order, payment), no_text: true %>
-            <% else %>
-              <%= link_to_with_icon action, Spree.t(action), fire_admin_order_payment_path(@order, payment, :e => action), :method => :put, :no_text => true, :data => {:action => action} %>
-            <% end %>
+            <%= link_to_with_icon action, Spree.t(action), fire_admin_order_payment_path(@order, payment, :e => action), :method => :put, :no_text => true, :data => {:action => action} %>
```
Comment (dkubb) [inline backend/app/views/spree/admin/payments/_list.html.erb]: You may want to use 1.9 style hashes here

File: backend/app/models/spree/backend_configuration.rb (hunk 1 above)
Comment (mbj) [inline backend/app/models/spree/backend_configuration.rb]: Extracted into: https://github.com/MountainRoseHerbs/spree/pull/225

**Pull request merged**